docs/source/basic_usage.rst

```diff
@@ -0,0 +1,55 @@
+.. _basic-usage:
+
+Basic Usage
```

Comment:
I suggest honing the focus of this page; it should describe making and using the schema file. How about calling it "Creating and Using a Data Resource Schema"? The headers will be:

* **Create a Data Resource Schema** (includes the content on the "Data Resource Schema Reference" page and the below "Utilities" section)
* **Use the Schema to Generate Data Resources**

I think "Interacting with Data Resources" should be a standalone page – it seems like it should be what you currently call "Routes."

How does this sound?